### ISSUE TITLE
Issues with pts/parboil-1.2.0 on Arch Linux

### ISSUE THREAD

**Issue by kingjdm** [phoronix-test-suite/test-profiles#52]
Hello,

Hopefully this is the right place to report this!

I've attempted to run some of the pts/parboil-1.2.0 tests but I keep experiencing  errors.

I've run: `phoronix-test-suite debug-benchmark parboil`

and it generates the following command: 

cd /home/<login>/.phoronix-test-suite/installed-tests/pts/parboil-1.2.0/ && ./parboil lbm omp_cpu long 2>&1

It then errors with the following after appearing to do a successful run

```
Traceback (most recent call last):
  File "tools/compare-output", line 8, in <module>
    import filecompare as fc
  File "/home/<login>/.phoronix-test-suite/installed-tests/pts/parboil-1.2.0/parboil-2.5-tree/common/python/filecompare.py", line 11
    raise TypeError, "Not a CompareMonad instance"
                   ^
SyntaxError: invalid syntax
Parboil parallel benchmark suite, version 0.2

Output checking tool detected a mismatch

The test quit with a non-zero exit status.


E: SyntaxError: invalid syntax
```
I'm wondering if this is some sort of python 2 vs 3 issue as with some of the other tests I get:

```
  File "tools/compare-output", line 12
    print "Pass"
               ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Pass")?
Parboil parallel benchmark suite, version 0.2

Output checking tool detected a mismatch

The test quit with a non-zero exit status.


E: SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Pass")?

```

On my system this is how python seems to resolve:

```
➜  .phoronix-test-suite python2 --version
Python 2.7.15
➜  .phoronix-test-suite python3 --version
Python 3.7.1
➜  .phoronix-test-suite python --version
Python 3.7.1

```

**Comment by kingjdm** [comment]
Did a little digging around and replaced the following line in a python script file like: `.phoronix-test-suite/installed-tests/pts/parboil-1.2.0/parboil-2.5-tree/benchmarks/lbm/tools/compare-output`

Before:
`#!/usr/bin/env python`

After:
`#!/usr/bin/env python2`

Is there a way for phoronix-test-suite to handle this behavior or would it need to be fixed upstream at the source?

**Comment by michaellarabel** [comment]
This issue should now be fixed by parboil-1.2.1. When python2 is present on the system, the test profile script will now just sed and replace the 'env python' references with 'env python2' to address the issue. 

Thanks for the report.

https://openbenchmarking.org/test/pts/parboil-1.2.1

**Comment by kingjdm** [comment]
Thanks for the help!
